src/nodes.coffee

```diff
@@ -1695,14 +1697,18 @@ exports.Code = class Code extends Base
     # parameters need to be sliced off the end of the splat parameter’s
     # array of arguments passed into the function.
     if paramsAfterSplat isnt 0
-      ref = new IdentifierLiteral splatParamName
-      val = new Call (new Value ref, [new Access new PropertyName 'slice']), [
-        new NumberLiteral 0
-        (new Op '-',
-          new Value(new IdentifierLiteral(splatParamName), [new Access new PropertyName 'length']),
-          new NumberLiteral paramsAfterSplat)
-      ]
-      exprs.push new Assign ref, val, '='
+      exprs.push new Assign new IdentifierLiteral(splatParamName), new Call (
+                                                                     new Value new IdentifierLiteral(splatParamName),
+                                                                     [new Access new PropertyName 'slice']),
+                                                                     [
+                                                                        new NumberLiteral 0
+                                                                        (new Op '-',
+                                                                          new Value(
+                                                                            new IdentifierLiteral(splatParamName),
+                                                                            [new Access new PropertyName 'length']
+                                                                          ),
+                                                                          new NumberLiteral paramsAfterSplat)
+                                                                     ]
```

Comment:
Personally, I always try to avoid horizontal alignment, because no matter how pedantic I try to be, it always goes out-of-sync after a while :) It also creates larger diffs than necessary, which increases the risk of merge conflicts (and who likes those?). But that's just my preference. I guess this is one of those things that can rapidly turn into a flame war :) That being said, I haven't really seen this style in the CoffeeScript code base before.